server/app/actions/ApplicantAuthorizationAction.java

```diff
@@ -75,7 +75,10 @@ private long getApplicantId(Request request, RouteExtractor routeExtractor) {
 
   /** Check if the profile is authorized to access the applicant's data. */
   private CompletionStage<Void> checkApplicantAuthorization(Request request, long applicantId) {
-    return profileUtils.currentUserProfile(request).orElseThrow().checkAuthorization(applicantId);
+    return profileUtils
+        .maybeCurrentUserProfile(request)
```

Comment:
(blocking) Please use `optional` instead of `maybe`

Reply:
Happy to make the change but not sure why - is there published guidance for this? There is tons of "maybe" in the codebase, and of the existing methods that return Optional there appears to be [one that starts with "maybe"](https://github.com/civiform/civiform/blob/50d86c656c8c2c92304b337388c59c7ef65e562d/server/app/services/LocalizedStrings.java#L179) and none that start with "optional" (based on code search for regex 'public Optional<[^>]+> maybe' and 'public Optional<[^>]+> optional').

Reply:
It's somewhere on the wiki. We decided a while ago to use `optional` and `optionally`. Maybe is legacy naming.

Reply:
Updated.